packages/dev/core/src/Animations/animatable.ts

```diff
@@ -1149,9 +1149,13 @@ Scene.prototype._processLateAnimationBindings = function (): void {
                     let startIndex = 0;
                     let normalizer = 1.0;
 
+                    const originalAnimationIsLoopRelative = originalAnimation && originalAnimation._animationState.loopMode === Animation.ANIMATIONLOOPMODE_RELATIVE;
+
                     if (holder.totalWeight < 1.0) {
                         // We need to mix the original value in
-                        if (originalAnimation && originalValue.scale) {
+                        if (originalAnimationIsLoopRelative) {
+                            finalValue = originalValue.clone();

```

Comment:
is it guaranteed that originalValue has a clone function in this case? I see that later it is being checked (`else if (originalValue.clone) {` )